pkg/util/util.go

```diff
@@ -1961,3 +1961,13 @@ func referSameObject(a, b metav1.OwnerReference) bool {
 
 	return aGV.Group == bGV.Group && a.Kind == b.Kind && a.Name == b.Name
 }
+
+// CheckNoobaaOwnerRefExistence receives an array of owner references and returns true if one of them is of a noobaa resource
+func CheckNoobaaOwnerRefExistence(ownerReferences []metav1.OwnerReference) bool {
```

Comment:
I suggest to use a cleared name, maybe something like this:
```suggestion
func isOwnedByNoobaa(ownerReferences []metav1.OwnerReference) bool {
```